salon_creator/lib/screens/home.dart

```diff
@@ -35,13 +38,102 @@ class _HomePageState extends State<HomePage> {
   }
 
   Widget _buildHomePageInternal(BuildContext context, UserModel userModel) {
+    final screenWidth = MediaQuery.of(context).size.width;
     return Scaffold(
       appBar: AppBar(
         title: Text('ホーム'),
         iconTheme: IconThemeData(color: Colors.black),
       ),
       drawer: _buildDrawer(context, userModel),
-      body: Container(),
+      body: SafeArea(
+        child: Column(
+          children: [
+            _buildScreenSelection(screenWidth),
+            selectedIndex == 0 ? LessonListScreen() : MemberListScreen(),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildScreenSelection(double screenWidth) {
+    return Container(
+      margin: EdgeInsets.only(
```

Comment:
Why not use padding? How is this different? If you do want to keep using margin, please consider defining a variable like marginValue

```
double marginValue = screenWidth * 0.02;
EdgeInsets.only(
        top: marginValue,
        bottom: marginValue,
        left: marginValue,
        right: marginValue,
      )
```